[DOC]Edit the SQS example README (#4335)
pachyderm/pachyderm: examples/spouts/SQS-S3/README.md

# Amazon SQS S3 Spout

This example describes how to create a simple spout
that listens for "object added" notifications on an
Amazon™ Simple Queue Service (SQS) queue, grabs the

> files, and places them in its output repo.

Reviewer: ```suggestion
files, and places them into a Pachyderm repo.
```

   ```

## Run the Spout

After you create an SQS spout, you can test it by uploading a file
into your S3 bucket and later finding it in the
SQS pipeline output repository.

To test the spout, complete the following steps:

1. In the IAM console, go to S3 and find your bucket.
1. Upload a file into your bucket. For example, `01-pipeline.png`. Depending
on the size of the file, it might take some time for the file to get uploaded.
1. In your terminal, run:

   ```bash
   $ pachctl list commit sqs-spout
   REPO      BRANCH COMMIT                           PARENT    STARTED        DURATION           SIZE
   sqs-spout master 4ecc933d523d485b8a9cce6b1feeac95 none      6 minutes ago  Less than a second 37.44KiB
   ```

1. Verify that the file that you have uploaded to the S3 bucket is
in the `sqs-spout` output repository. Example:

   ```bash
   $ pachctl list file sqs-spout@master
   NAME             TYPE SIZE
   /01-pipeline.png file 37.44KiB
   ```

## Pipeline Environment Parameters

This table describes pipeline parameters that you can specify in your
pipeline specification.

| Optional Parameter  | Description   |
| ------------------- | ------------- |

> | `-h`, `--help`          | Show help message and exit. |

Reviewer: including help doesn't really make sense. every command has help, but its not a meaningful optional argument to include here IMO
